app/menu.js

```diff
@@ -393,18 +392,17 @@ const init = (settingsState, args) => {
           accelerator: 'Shift+CmdOrCtrl+Delete',
           enabled: siteUtil.hasNoTagSites(appStore.getState().get('sites')),
```

Comment:
I think the menu needs to be rebuilt when the first tagless site is added; otherwise this shows up as disabled even though i have visited some sites.
STR:
clear history
visit a site
open History menu, the 'clear history' item is disabled